README.md

```diff
@@ -1,6 +1,15 @@
 # TasteIT
-A recipe bank with a scaling function.
+A recipe bank for the IT-section in Chalmers.
 
-This repo mostly contains `good first issues` which makes it a good starting point for any one interested in helping out digIT develop their IT services.
+## How to start the webpage
+1. Start by cloning down the project in a separate fork or branch
```

Comment:
Maybe just "Clone down the project" because every step starts with start (ugly) and you don't need a separate branch or fork to run the webpage

Reply:
Updated now to just say _Clone or download the project_.